### PR TITLE
Make ECS instance storage size configurable use latest AMI

### PR DESCRIPTION
**Pull request by jonnyshaw89**
The PR allows us to increase the root storage size of the ECS instances.

I have also changed the way we lookup the ECS instance AMI id so that we always launch the latest version of the AMI.

To test this change your `eq-ecs` module to point at this branch in `eq-terraform` by appending `?ref=ecs-instance-updates` to the end of the source

### CODE REVIEW

File: global_vars.tf [hunk 1]
```diff
@@ -21,6 +21,11 @@ variable "ecs_instance_type" {
   default     = "t2.medium"
 }
 
+variable "ecs_instance_storage_size" {
+  description = "The root block storage size of the EC2 instance"
```
Comment (ajmaddaford) [inline global_vars.tf]: It would be useful to also add "in GB" to the description

File: ecs.tf [hunk 2]
```diff
@@ -10,15 +10,34 @@ data "template_file" "ecs_user_data" {
...
+
+  filter {
+    name   = "owner-alias"
+    values = ["amazon"]
+  }
+
+  name_regex = ".+-amazon-ecs-optimized$"
+
+  most_recent = true
+}
```
Comment (ajmaddaford) [inline ecs.tf]: We should either move the ami changes into another pull request, or at least split it out into another commit. If we do keep it in this pull request, the title needs changing.

Comment (jonnyshaw89) [reply ecs.tf]: Done
